WS2012R2/lisa/remote-scripts/ica/CORE_StressReloadModules.sh

```diff
@@ -146,22 +146,24 @@ if [ $? -eq 0 ]; then
     exit 1
 fi
 
+# if not set the LoopCount, set as 2000 by default
+if [ "${LoopCount:-UNDEFINED}" = "UNDEFINED" ]; then
+    LoopCount=2000
+fi
+msg="Info: module unload/load loop count set as $LoopCount"
+LogMsg "${msg}"
+echo "$msg" >> ~/summary.log
+
 pass=0
 START=$(date +%s)
-while [ $pass -lt 100 ]
+while [ $pass -lt $LoopCount ]
 do
     modprobe -r hv_netvsc
-    sleep 1
```

Comment:
we should keep the sleep commands in here, in the past it was discussed that a short delay must exist in the unload/load of the module. it should also not be a real-world scenario where this would occur instantly to unload/load.